### PR TITLE
Direct ingestion/Delta support

### PR DESCRIPTION
**Pull request by vikramraman**

### CODE REVIEW

File: test/tests.go [hunk 1]
```diff
@@ -2,13 +2,24 @@ package main
 
 import (
 	"fmt"
-	"net"
-	"time"
+	//"net"
```
Comment (panghy) [inline test/tests.go]: remove?

**Comment by panghy** [comment]
Can you also add a continuous build on travis?

**Comment by vikramraman** [comment]
@panghy Will do (re: travis builds). Thanks.

**Comment by vikramraman** [comment]
@panghy Added travis-ci builds for this repo.

**Pull request merged**